src/main/java/com/puppycrawl/tools/checkstyle/Checker.java

```diff
@@ -133,6 +133,9 @@
     /** Controls whether exceptions should halt execution or not. */
     private boolean haltOnException = true;
 
+    /** If true old style force loading will be used. */
+    private boolean forceLoad;
```

Comment:
As discussed at https://github.com/checkstyle/checkstyle/pull/5245#issuecomment-343007369 , this property should be in `PackageObjectFactory` and not in `Checker`.